src/test/resources/testdata/objects/capability_object.json

```diff
@@ -42,7 +42,8 @@
         8,
         9,
         10
-      ]
+      ],
+      "banks": []
```

Comment:
Can we also have the test cases where `banks` is not empty? So, we can ensure that `Bank` model is serialized correctly.